.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+name: test
+
+on: workflow_dispatch
```

You can add this to run on PRs so we get the feedback.
When we turn to focus on official support we'll add more scenarios to ensure coverage on release and push also
```suggestion
on: 
  pull_request:
    branches: [main]
  workflow_dispatch
```